Formula/guile.rb

```diff
@@ -37,9 +37,7 @@ class Guile < Formula
   depends_on "pkg-config" # guile-config is a wrapper around pkg-config.
   depends_on "readline"
 
-  on_linux do
-    depends_on "gperf"
-  end
+  uses_from_macos "gperf"
```

Comment:
This caused an audit check failure.

```
  * 40: col 3: FormulaAudit/UsesFromMacos: `uses_from_macos` should only be used for macOS dependencies, not gperf.
```

Reply:
I already did it, and it was released in brew 2.7.2: https://github.com/Homebrew/brew/pull/10207

Weird